src/javascripts/ng-admin/Crud/column/maColumn.js

```diff
@@ -3,7 +3,22 @@
 define(function (require) {
     'use strict';
 
-    function maColumn($location, $anchorScroll, $compile, Configuration) {
+    var _ = require('lodash');
```

Comment:
Well, we need to because we `map` on an object, not an array, and ES5 doesn't know how to do that.